src/modules/localization/domain/kalman_filter/Dynamics.cpp

```diff
@@ -0,0 +1,40 @@
+#include "Dynamics.hpp"
```

Comment:
nit: #include <> instead of #include ""

Does not make a difference. Just a convention

Reply:
Doesn't <> this is used for librarys defined globally and "" this generally forces compiler to check in the folder directly first then in its global library which is better for custom defined library

Reply:
I dont know much about this stuff but from what I googled, <> searches in directories that are included in compiler path. 
My guess is that when we do colcon build, the include files get automatically included in this path list so no need to do "" (unless we do not know what we are doing).

Performance wise there is no difference at all.